selfplay_mcts.py

```diff
@@ -39,7 +39,7 @@ def play(network, readouts, resign_threshold, verbosity=0):
 
     # Disable resign in 5% of games
     if random.random() < 0.05:
-        player.resign_threshold = -0.9999
+        player.resign_threshold = -0.999999
```

Comment:
add to hyperparams?

Reply:
ack

Reply:
why don't we just make this -1?

Reply:
Originally, i was ok with it resigning if it literally only has -1's at the foot of the tree.  I'm a bit more confident it'll do the right thing now :)